4.1/contrib/docker-compose-planet.yml

```diff
@@ -18,9 +18,9 @@ services:
             PBF_URL: https://ftp5.gwdg.de/pub/misc/openstreetmap/planet.openstreetmap.org/pbf/planet-latest.osm.pbf
             REPLICATION_URL: https://ftp5.gwdg.de/pub/misc/openstreetmap/planet.openstreetmap.org/replication/day/
             NOMINATIM_PASSWORD: very_secure_password
-            IMPORT_WIKIPEDIA: true
```

Comment:
any specifics why we use quotes for this boolean values now, and not for the password etc.?

Reply:
boolean is invalid. 

```bash
$ docker-compose -f docker-compose-planet.yml config 
ERROR: The Compose file './docker-compose-planet.yml' is invalid because:
services.nominatim.environment.IMPORT_WIKIPEDIA contains true, which is an invalid type, it should be a string, number, or a null
```